lib/utils/razorpay-utils.d.ts

```diff
@@ -0,0 +1,56 @@
+export interface IRazorpayWebhook {
+    payment_id: string;
+}
+
+export interface IRazorpayVerifyPayment extends IRazorpayWebhook {
+    order_id: string;
+}
+
+export interface IRazorpayVerifySubscription extends IRazorpayWebhook {
+    subscription_id: string;
+}
+
+export interface IRazorpayVerifyPaymentLink extends IRazorpayWebhook {
+    payment_link_id: string;
+    payment_link_reference_id: string;
+    payment_link_status: string;
+}
+
+/**
+* Verify webhook verification
+*
+* @param {string} body 
+* raw webhook request body
+* @param {string} signature  
+* The hash signature is calculated using HMAC with SHA256 algorithm; with your webhook 
+* secret set as the key and the webhook request body as the message.
+* @param {string} secret
+* your webhook secret
+*
+*/
+export function validateWebhookSignature(body: string, signature: string, secret: string): boolean
+
+/**
+*  Payment verfication
+*
+* @param {object} payload
+* Check [doc](https://github.com/razorpay/razorpay-node/blob/master/documents/paymentVerfication.md) for required params
+* @param {string} signature
+* The hash signature is calculated using HMAC with SHA256 algorithm; with your webhook 
+* secret set as the key and the webhook request body as the message.
+* @param {string} secret
+* your webhook secret
+*
+*/
+export function validatePaymentVerification(payload: IRazorpayVerifyPayment, signature: string, secret: string): boolean
+export function validatePaymentVerification(payload: IRazorpayVerifySubscription, signature: string, secret: string): boolean
+export function validatePaymentVerification(payload: IRazorpayVerifyPaymentLink, signature: string, secret: string): boolean
```

Comment:
These 3 overloads can be instead changed to use union since validatePaymentVerification function doesn't return anything different based on the `payload` param

```ts
export function validatePaymentVerification(payload: IRazorpayVerifyPayment | IRazorpayVerifySubscription | IRazorpayVerifyPaymentLink, signature: string, secret: string): boolean
```